terraform/modules/monitoring/certs.tf

```diff
@@ -1,9 +1,23 @@
-resource "aws_route53_zone" "public" {
-  name = local.workspace_external_domain
+data "aws_route53_zone" "public" {
+  name = var.external_app_domain
 }
 
-resource "aws_acm_certificate" "public" {
-  domain_name = "*.${local.workspace_external_domain}"
+resource "aws_route53_record" "monitoring_public_zone_ns" {
+  zone_id = data.aws_route53_zone.public.zone_id
```

Comment:
Would this create a record into a zone created by the `govuk-publishing-platform` deployment?

If so, would that prevent us from destroying the zone without first destroying this record?

Reply:
Is it just ecs.test.govuk.digital that we want to be able to destroy nightly or also test.govuk.digital?

I think this PR basically shifts the issue one level up the DNS hierarchy (which might solve it, if we don't need to tear down test.govuk.digital).

Reply:
no, it will not. it will create `monitoring.test.govuk.digital` with `test.govuk.digital` being created in govuk-aws.